ostrica/Plugins/BlackLists/__init__.py

```diff
@@ -21,13 +21,25 @@
 #				along with OSTrICa. If not, see <http://www.gnu.org/licenses/>.
 #-------------------------------------------------------------------------------
 import sys
-import httplib
-import string
-import socket
+if sys.version_info < (3, 0):
+  import httplib
+  import StringIO
+else:
+  import http.client as httplib
+  import io as StringIO
 import gzip
-import re
-import StringIO
-from bs4 import BeautifulSoup
+
+def get_zip_obj(data):
+  if sys.version_info < (3, 0):
+    return StringIO.StringIO(data)
+  else:
+    return StringIO.BytesIO(data)
+  return data
```

Comment:
I will fix this by moving the code from all places in the scripts to a general `lib/toolkit.py` library. It's a good way to make functions available in multiple scripts (e.g all the plug-ins) while making it easy to maintain (only one location)